nbviewer/utils.py

```diff
@@ -8,6 +8,7 @@
 import base64
 import cgi
 import re
+from collections import OrderedDict
```

Comment:
There be dragons here.  I have another change in the pipeline that will get the global state out of this module.